docs/accessibility/content.md

```diff
@@ -157,9 +157,42 @@ Microcopy comprises short words, phrases, or chunks of text used on the web or i
 
 Best practices for all microcopy:
 
-- Keep text short but descriptive. Use as few words as possible, but avoid non-informative or vague text, like “click here,” “learn more,” “read more,” etc.
-- Make the first word meaningful. For example, avoid starting with “a”, “the”, “for”, etc. In many cases, this practice allows screen reader users to navigate items alphabetically.
-- Keep the text consistent. Ensure that consistency is maintained on pages, sites, and entire user journeys.
+- Keep text short but descriptive
+- Use as few words as possible
+- Avoid non-informative or vague text, like “click here,” “learn more,” “read more,” etc.
+- Make the first word meaningful
+    - For example, avoid starting with “a”, “the”, “for”, etc
+    - In many cases, this practice allows screen reader users to navigate items alphabetically
+- Keep the text consistent.
+    - Ensure that consistency is maintained on pages, sites, and entire user journeys.
+
+### Creating emphasis
+
+When writing for the web, use bold and italic formatting sparingly. Remember: if everything is emphasized, nothing is emphasized.
+
+Red Hat design guidelines mandate users employ either bold or italics—never both together.
+
+#### Bolding
```

Comment:
Maybe make the headings consistent as verbs or nouns?

- Bolding / Bold
- Italicizing / Italics
- Underlining / Underlines